apps/website/src/utils.ts

```diff
@@ -1,2 +1,5 @@
 export const drupalUrl =
   process.env.PUBLIC_DRUPAL_URL || 'http://127.0.0.1:8888';
+
+export const frontendUrl =
+  process.env.WAKU_PUBLIC_URL || 'http://127.0.0.1:8000';
```

Comment:
I hope with devbox we will be able to get rid of `ENV_VAR || 'default'` in the code 🤞